canonical/charmed-kubeflow.io
# WD-12460 New Documentation Layout

## Done


## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8046
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check if the /docs page is working fine and you are able to see a right side navigation panel with the headings of the document.
- Check if the header, footer and search are well alinged, use [multipass](https://multipass.run/docs) as a reference for the layout.
- Here is the [demo](https://charmed-kubeflow-io-179.demos.haus/docs) generated for the PR.

## Issue / Card

Fixes # [WD-12460](https://warthogs.atlassian.net/browse/WD-12460)





[WD-12460]: https://warthogs.atlassian.net/browse/WD-12460?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
